src/app.jsx

```diff
@@ -494,6 +494,35 @@ const NavigatorCardBody = ({
                 });
             } else if (e.key === "Enter" && selected.length === 1) {
                 onDoubleClickNavigate(selected[0]);
+            } else if (e.key === "Delete") {
+                if (selected.length !== 0) {
+                    const currentDir = path.join("/") + "/";
+                    if (selected.length === 1) {
+                        deleteItem(
+                            Dialogs,
+                            {
+                                selected,
+                                itemPath: currentDir + selected[0].name,
+                                setHistory,
+                                setHistoryIndex,
+                                path: currentDir,
+                                setSelected
+                            }
+                        );
+                    } else {
+                        deleteItem(
+                            Dialogs,
+                            {
+                                selected,
+                                itemPath: currentDir + selected.map(s => currentDir + s.name),
+                                setHistory,
+                                setHistoryIndex,
+                                path: currentDir,
+                                setSelected
+                            }
```

Comment:
Both conditionals are almost the same, and can be combined into one as selected is always an array.